### PR TITLE
add utility functions NmtBin.get_ell_min/max

### PR DESCRIPTION
**Pull request by joezuntz**
This adds two minor functions that I've found useful in TXPipe.

### CODE REVIEW

File: test/test_nmt_bins.py [hunk 1]
```diff
@@ -62,6 +62,13 @@ def test_bins_edges(self):
         self.assertTrue(np.sum(np.fabs(self.bc.get_effective_ells() -
                                        self.be.get_effective_ells())) < 1E-10)
 
+    def test_min_max(self):
+        n = self.get_n_bands()
```
Comment (damonge) [inline test/test_nmt_bins.py:66]: I think you want `self.be.get_n_bands()` here

**Review by damonge — CHANGES_REQUESTED** [review]
Thanks a lot @joezuntz 
Tests seem to be failing because of a typo, see below.

File: test/test_nmt_bins.py (hunk 1 above)
Comment (joezuntz) [reply test/test_nmt_bins.py]: Thanks - I couldn't face compiling on my laptop and messing with my current install, so thought I'd leave it to travis to run the tests.

**Comment by joezuntz** [comment]
The last round of tests failed due to some kind of build error before it got to the tests.  Do you have the power to just re-run it?  Otherwise I can just do a no-op commit.

**Comment by damonge** [comment]
Frustrating. Yep, just did. If it doesn't pass I can just clone your branch and fix

File: setup.py [hunk 2]
```diff
@@ -73,22 +73,22 @@ def _compile_libnmt(): # RM
 class build(_build): # RM
     """Specialized Python source builder.""" # RM
     def run(self): # RM
-        _compile_libsharp() # RM
-        _compile_libchealpix() # RM
+        # _compile_libsharp() # RM
+        # _compile_libchealpix() # RM
```
Comment (damonge) [inline setup.py:77]: ah please uncomment these two

File: setup.py [hunk 3]
```diff
@@ -73,22 +73,22 @@ def _compile_libnmt(): # RM
...
         _compile_libnmt() # RM
         _build.run(self) # RM
 
 class develop(_develop): # RM
     """Specialized Python develop mode.""" # RM
     def run(self): # RM
-        _compile_libsharp() # RM
-        _compile_libchealpix() # RM
+        # _compile_libsharp() # RM
+        # _compile_libchealpix() # RM
```
Comment (damonge) [inline setup.py:85]: and these two

File: setup.py [hunk 4]
```diff
@@ -73,22 +73,22 @@ def _compile_libnmt(): # RM
...
         _compile_libnmt() # RM
         _build.run(self) # RM
 
 _nmtlib = Extension("_nmtlib",
                     ["pymaster/namaster_wrap.c"],
-                    extra_objects=["./_deps/lib/libnmt.a", "./_deps/lib/libchealpix.a"],
+                    extra_objects=["./_deps/lib/libnmt.a"],#, "./_deps/lib/libchealpix.a"],
```
Comment (damonge) [inline setup.py:91]: And this

**Review by damonge — CHANGES_REQUESTED** [review]

**Comment by joezuntz** [comment]
Crap, sorry, the changes in the setup file weren’t supposed to be included.

**Comment by joezuntz** [comment]
I've undone the changes to the setup.py file.  Let's see now if the tests run properly.

**Review by damonge — APPROVED** [review]
OK, great! Thanks a lot!

**Pull request merged**